driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudTest.java

```diff
@@ -25,13 +25,21 @@
 import java.net.URISyntaxException;
 import java.util.Collection;
 
+import static org.junit.Assume.assumeFalse;
+
 public class UnifiedCrudTest extends UnifiedSyncTest {
 
     public UnifiedCrudTest(@SuppressWarnings("unused") final String fileDescription,
                            @SuppressWarnings("unused") final String testDescription,
                            final String schemaVersion, @Nullable final BsonArray runOnRequirements, final BsonArray entities,
                            final BsonArray initialData, final BsonDocument definition) {
         super(schemaVersion, runOnRequirements, entities, initialData, definition);
+        assumeFalse(testDescription.equals("Unacknowledged findOneAndReplace with hint string on 4.4+ server"));
```

Comment:
Given the class structure there is currently no straightforward way to share these assumptions.  I'm thinking about opening a JAVA ticket to address this shortcoming.